src/components/FilterMenu.jsx

```diff
@@ -0,0 +1,256 @@
+                      </MenuItemOption>
+                    ))}
+                  </MenuOptionGroup>
+                  {/* Certifications */}
+                  <MenuOptionGroup title="Certifications" type="checkbox">
+                    {certificationOptions.map((option) => (
+                      <MenuItemOption
+                        key={`certifications${option.id}`}
+                        value={option.value}
+                        onClick={() =>
+                          handleFilterClick(option.label, "Certifications")
+                        }
+                      >
+                        {option.label}
+                      </MenuItemOption>
+                    ))}
+                  </MenuOptionGroup>
+                </Flex>
+              </Wrap>
+              {/* Salary Range Slider */}
+              <Flex justifyContent="center">
+                <Box width="50%" ml={6} bottom={4}>
+                  <Heading size="sm">Salary Range</Heading>
+                  <Flex alignItems="center">
+                    <Text fontSize="sm" mr="2">
+                      $0
+                    </Text>
+                    <Slider
+                      flex="1"
+                      aria-label="salary range slider"
+                      defaultValue={salaryRange}
+                      onChange={handleSalaryChange}
+                      min={0}
+                      max={300000} //Maximum salary range
+                      onTouchStart={(e) => e.preventDefault()}
+                      onTouchMove={(e) => e.preventDefault()}
+                    >
+                      <SliderTrack>
+                        <SliderFilledTrack />
+                      </SliderTrack>
+                      <SliderThumb />
+                    </Slider>
+                    <Text fontSize="sm" ml="2">
+                      ${salaryRange}
+                    </Text>
+                  </Flex>
+                </Box>
+              </Flex>
+              {/* Industry */}
+              <Wrap spacing="4px">
+                <Flex
+                  flexWrap="wrap"
+                  direction={isLargerThanSmall ? "row" : "column"}
+                >
+                  {/* Industry options split into two columns */}
+                  <Box flex="1">
+                    <MenuOptionGroup title="Industry" type="checkbox">
+                      {industryOptions
+                        .slice(0, Math.ceil(industryOptions.length / 2))
+                        .map((option) => (
+                          <MenuItemOption
+                            key={`industry${option.id}`}
+                            value={option.value}
+                            onClick={() =>
+                              handleFilterClick(option.label, "Industry")
+                            }
+                          >
+                            {option.label}
+                          </MenuItemOption>
+                        ))}
```

Comment:
This code looks almost identical to the block on line 231, this could probably be abstracted into a function to make it more DRY